src/test/java/com/example/java/test/junior/developer/question/QuestionMapperJunitTest.java

```diff
@@ -0,0 +1,42 @@
+package com.example.java.test.junior.developer.question;
+
+import com.example.java.test.junior.developer.dto.QuestionDto;
+import com.example.java.test.junior.developer.mapper.QuestionMapper;
+import com.example.java.test.junior.developer.model.Question;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+public class QuestionMapperJunitTest {
+
+    @InjectMocks
```

Comment:
What is the purpose of this annotation? Does your mapper has dependencies on any other classes so that you need to mock it?